src/sqlfluff/dialects/dialect_bigquery.py

```diff
@@ -202,7 +202,7 @@
         Bracketed(
             Sequence(
                 Ref("DatatypeSegment"),
-                Bracketed(Anything(), optional=True),  # For types like STRING(10)
```

Comment:
Finally deleted the comment. STRING(10) is no longer a good example, as I made it Delimited because I needed to support the NUMERIC(5, 2) format. On the other hand, setting the example to NUMERIC(5, 2) is misleading in relation to "Numeric"LiteralSegment.